core/org.wso2.carbon.ui/src/main/resources/web/dialog/js/dialog.js

```diff
@@ -286,6 +286,90 @@ CARBON.showPopupDialog = function(message, title, windowHight, okButton, callbac
     }
 };
 
+/**
+ * Display any info inside a jQuery UI's confirmation widget.
+ * @method showPopupDialog
+ * @param {String} message to display html/text
+ * @return {Boolean}
+ */
+CARBON.showPopupConfirm = function(htmlmessage, title, windowHeight, okButton, callback, windowWidth) {
```

Comment:
shall we have width next to height? and shall we introduce two strings which can override 'OK' and 'Cancel'